Sources/Extensions/NSDictionary+Toggles.swift

```diff
@@ -0,0 +1,39 @@
+//  NSDictionary+Toggles.swift
+
+import Foundation
+
+extension NSDictionary {
+    public enum ToggleConversionError: Error {
+        case nsDictionaryInvalidFormat
+        case nsDictionaryEmpty
+    }
+    
+    public func convertToToggleDataSource() throws -> [Variable: Toggles.Value] {
+        guard let swiftDictionary = self as? [String: Any], !swiftDictionary.isEmpty else {
```

Comment:
No need to attempt a cast to `[String: Any]` if we use an extension on Swift dictionary w/ `Key == String, Value == Any`.